Bugzilla/Template.pm

```diff
@@ -398,7 +398,13 @@ sub css_files {
 
     unshift @requested_css, "skins/yui.css" unless $no_yui;
 
-    my @css_sets = map { _css_link_set($_) } @requested_css;
+    my @css_sets = map { _css_link_set($_) } sort {
+        my $first_a = $a =~ m!js/jquery!;
+        my $first_b = $b =~ m!js/jquery!;
```

Comment:
as this could be called a lot (because sort) it would be better to use index() for a substring search